eZ/Publish/Core/Repository/Mapper/ContentDomainMapper.php

```diff
@@ -171,7 +171,7 @@ public function buildContentProxy(
      * @param string[] $prioritizedLanguages
      * @param bool $useAlwaysAvailable
      *
-     * @return \eZ\Publish\API\Repository\Values\Content\Content[<int>]
+     * @return \eZ\Publish\API\Repository\Values\Content\Content [<int>]
```

Comment:
```suggestion
     * @return \eZ\Publish\API\Repository\Values\Content\Content[]
```
anything else is not really supported